### PR TITLE
[terra-functional-testing] Update to use seleniarm 4.10 docker images for Apple silicon support

### PR DESCRIPTION
**Pull request by artpark**
### Summary

**What was changed:**
The docker compose file now pulls in the seleniarm-4.10 image. This will be considered a major version change as the screenshots generated by the new docker image is different. This is because the chrome version in the image is higher -- which comes with updated styling changes.

**Why it was changed:**
The previous docker image was not compatible with Apple silicon chips.

### Testing

This change was tested using:

- [x] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [x] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-9173 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-functional-testing/src/docker/docker-compose.yml [hunk 1]
```diff
@@ -1,43 +1,36 @@
...
       - /dev/shm:/dev/shm
+    shm_size: 2gb
     depends_on:
       - selenium-hub
     environment:
-      - HUB_HOST=selenium-hub
-      - HUB_PORT=4444
+      - SE_EVENT_BUS_HOST=selenium-hub
+      - SE_EVENT_BUS_PUBLISH_PORT=4442
+      - SE_EVENT_BUS_SUBSCRIBE_PORT=4443
```
Comment (sdadn) [inline packages/terra-functional-testing/src/docker/docker-compose.yml]: nit: newline error
```suggestion
      - SE_EVENT_BUS_SUBSCRIBE_PORT=4443

```

File: packages/terra-functional-testing/src/docker/docker-compose.yml [hunk 2]
```diff
@@ -1,43 +1,36 @@
 version: "3"
-
 services:
   selenium-hub:
-    image: selenium/hub:${TERRA_SELENIUM_DOCKER_VERSION:-3.141.59-zirconium}
```
Comment (sdadn) [inline packages/terra-functional-testing/src/docker/docker-compose.yml:5]: Were you able to find out what `TERRA_SELENIUM_DOCKER_VERSION` is?

**Comment by sdadn** [comment]
The overall PR looks good! Before we merge, can we test this with terra-core?

File: packages/terra-functional-testing/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 * Changed
   * Updated `uuid` to `v8.2.0` for consistency across Terra repos.
```
Comment (sdadn) [inline packages/terra-functional-testing/CHANGELOG.md:6]: Let's release this change before we merge the MVB into main.

File: packages/terra-functional-testing/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,7 @@
...
+  * Updated to pull in the seleniarm 4.10 docker images.
```
Comment (sdadn) [inline packages/terra-functional-testing/CHANGELOG.md]: Let's move the changelog entry under a Breaking Changes section:
```suggestion
* Breaking Changes
  * Updated to pull in the seleniarm 4.10 docker images.

* Changed
  * Updated `uuid` to `v8.2.0` for consistency across Terra repos.

```

File: packages/terra-functional-testing/CHANGELOG.md [hunk 5]
```diff
@@ -2,9 +2,15 @@
...
 
-* Changed
+* Breaking changes
+  * Updated to pull in the seleniarm 4.10 docker images.
+
+* Changes
   * Updated `uuid` to `v8.2.0` for consistency across Terra repos.
 
+* Added
+  * Added `useHttps` to use a secure http connection when the `gridUrl` flag is also provided.
```
Comment (sdadn) [inline packages/terra-functional-testing/CHANGELOG.md]: What is the purpose of this change?

Comment (artpark) [reply packages/terra-functional-testing/CHANGELOG.md]: This was in response to this request from Binwei: https://jira2.cerner.com/browse/ION-47621?focusedCommentId=12668119&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-12668119

I'll update our jira and pr to reflect that this was added in

**Comment by artpark** [comment]
> The overall PR looks good! Before we merge, can we test this with terra-core?

Tested with terra-core, everything looks good

**Review by sdadn — APPROVED** [review]
